include/GafferScene/FilterPlug.h

```diff
@@ -100,8 +100,8 @@ class GAFFERSCENE_API FilterPlug : public Gaffer::IntPlug
 		struct SceneScope : public Gaffer::Context::EditableScope
 		{
 			SceneScope( const Gaffer::Context *context, const ScenePlug *scenePlug );
-		private:
-			const ScenePlug *m_scenePlug;
+			private :
+				const ScenePlug *m_scenePlug;
```

Comment:
I guess maybe we have a special convention for structs that I'm not familiar with since we don't have any structs, but it looks very odd to my eye to have the constructor at a different indentation to other member functions, and I don't think I've ever seen this in any C++ convention.

Reply:
It's a pretty simple application of our formatting rules - the contents of a struct/class are indented, and the `private/public` marker adds another level of indentation. I wonder, do you find this objectionable for the `Context::Value` class I added as well (you didn't comment there)? Personally I actually find it improves clarity - the public stuff is "right there" and the private stuff is a little more out of the way.